src/Microsoft.CodeAnalysis.Analyzers/UnitTests/MetaAnalyzers/DiagnosticDescriptorCreationAnalyzerTests.cs

```diff
@@ -3553,6 +3722,177 @@ End Class
                 GetRS1007ExpectedDiagnostic(3));
         }
 
+        [Fact, WorkItem(3958, "https://github.com/dotnet/roslyn-analyzers/issues/3958")]
+        public async Task RS1032_LeadingOrTrailingWhitespaces_Diagnostic()
+        {
+            var additionalFileName = "Resources.resx";
+            var additionalFileText = @"
+<root>
+  <data name=""AnalyzerMessage1"" xml:space=""preserve"">
+    <value>Message with trailing whitespace </value>
```

Comment:
Can you add another scenario where trailing spaces are after a period, similar to the case caught in the original PR, say `Message with trailing whitespaces after a period.   `? I have a feeling your current logic might be need to be tweaked to move whitespace fixing part above multi-line fixing part.